src/CreateRecipe/CreateRecipe.jsx

```diff
@@ -29,7 +25,7 @@ export default function CreateRecipe() {
           name: recipeName.value,
           description: description.value,
           cookingTime: cookingTime.value,
-          ingredients: ingredients.value,
+          ingredients: ingredient,
```

Comment:
Would be prettier if both values are plural or singular `ingredients: ingredients` or `ingredient: ingredient` but thats just a preference really.